### PR TITLE
Location overview

### PR DESCRIPTION
**Pull request by KatBirk**
Added ability to see locations for the public. Added ability to create, edit and delete locations for people with locations_crud permission

### CODE REVIEW

File: app/Http/Controllers/LocationController.php [hunk 1]
```diff
@@ -2,9 +2,126 @@
...
+            'long_description'=>'string',
+            'name' => 'required|string|min:2',
+            'cover_img_path'=>'nullable|image|mimes:jpeg,png|max:2048',
+            'street_number' =>'string|required',
+            'street_name' => 'string|required',
+            'zip_code'=> 'string|required',
+            'city'=>'string|required',
+            'country'=>'string|required'
+        ]);
+        $address = Address::create([
```
Comment (Darchie4) [inline app/Http/Controllers/LocationController.php:39]: What if address already exists in database?
Like if a single address have multiple danse sale

Hint: Du kan bruge "firstOrCreate()"

File: app/Http/Controllers/LocationController.php [hunk 2]
```diff
@@ -2,9 +2,126 @@
...
+            'name' => 'required|string|min:2',
+            'cover_img_path'=>'nullable|image|mimes:jpeg,png|max:2048',
+            'street_number' =>'string|required',
+            'street_name' => 'string|required',
+            'zip_code'=> 'string|required',
+            'city'=>'string|required',
+            'country'=>'string|required'
+        ]);
+
+        $address->update([
```
Comment (Darchie4) [inline app/Http/Controllers/LocationController.php:88]: Same problem here, what if multiple rooms were linked to this address, we should probably just make a new one.

Men det her er nok en smags sag...

**Review by Darchie4 — DISMISSED** [review]
Looks good, Just small oversights

File: app/Http/Controllers/LocationController.php (hunk 2 above)
Comment (KatBirk) [reply app/Http/Controllers/LocationController.php]: Er rettet

File: app/Http/Controllers/LocationController.php (hunk 1 above)
Comment (KatBirk) [reply app/Http/Controllers/LocationController.php]: Er rettet

**Comment by KatBirk** [comment]
Alle change requests are rettet. Please review changes.

**Pull request merged**